src/common/models/Projection.ts

```diff
@@ -485,19 +484,31 @@ export class Projection {
         dates: [],
       };
     }
-    let earliestDate, latestDate;
+    let earliestDate, latestDate, year, month, date;
     // If we have projections, we use that time range; else we use the actuals.
     // TODO(chris): Is there a reason that this was bound to the projections timeseries first?
     // It cuts off some of the earlier dates
     if (metricsTimeseriesRaw.length > 0) {
-      earliestDate = moment.utc(first(metricsTimeseriesRaw)!.date);
-      latestDate = moment.utc(last(metricsTimeseriesRaw)!.date);
+      [year, month, date] = first(metricsTimeseriesRaw)!
+        .date.split('-')
+        .map(elem => parseInt(elem));
+      earliestDate = new Date(Date.UTC(year, month - 1, date));
+      [year, month, date] = last(metricsTimeseriesRaw)!
+        .date.split('-')
+        .map(elem => parseInt(elem));
+      latestDate = new Date(Date.UTC(year, month - 1, date));
     } else {
-      earliestDate = moment.utc(first(actualsTimeseriesRaw)!.date);
-      latestDate = moment.utc(last(actualsTimeseriesRaw)!.date);
+      [year, month, date] = first(actualsTimeseriesRaw)!
+        .date.split('-')
+        .map(elem => parseInt(elem));
+      earliestDate = new Date(Date.UTC(year, month - 1, date));
+      [year, month, date] = last(actualsTimeseriesRaw)!
+        .date.split('-')
+        .map(elem => parseInt(elem));
+      latestDate = new Date(Date.UTC(year, month - 1, date));
```

Comment:
If we do need to do this complicated string parsing / date creation dance, we should move it to a helper function so it's not duplicated 4 times. 😁